metron-interface/metron-rest/README.md

```diff
@@ -131,11 +131,11 @@ Request and Response objects are JSON formatted.  The JSON schemas are available
 | [ `GET /api/v1/storm/indexing/deactivate`](#get-apiv1stormindexingdeactivate)|
 | [ `GET /api/v1/storm/indexing/start`](#get-apiv1stormindexingstart)|
 | [ `GET /api/v1/storm/indexing/stop`](#get-apiv1stormindexingstop)|
-| [ `GET /api/v1/storm/parser/activate/{name}`](#get-apiv1stormparseractivate{name})|
-| [ `GET /api/v1/storm/parser/deactivate/{name}`](#get-apiv1stormparserdeactivate{name})|
-| [ `GET /api/v1/storm/parser/start/{name}`](#get-apiv1stormparserstart{name})|
-| [ `GET /api/v1/storm/parser/stop/{name}`](#get-apiv1stormparserstop{name})|
-| [ `GET /api/v1/storm/{name}`](#get-apiv1storm{name})|
+| [ `GET /api/v1/storm/parser/activate/{name}`](#get-apiv1stormparseractivatename)|
+| [ `GET /api/v1/storm/parser/deactivate/{name}`](#get-apiv1stormparserdeactivatename)|
+| [ `GET /api/v1/storm/parser/start/{name}`](#get-apiv1stormparserstartname)|
+| [ `GET /api/v1/storm/parser/stop/{name}`](#get-apiv1stormparserstopname)|
+| [ `GET /api/v1/storm/{name}`](#get-apiv1stormname)|
```

Comment:
I'd like to propose a patch to fix these up.  As you've probably noted, there's already logic that re-writes the links based on the link label, but the syntax here exceeds what was anticipated.